### PR TITLE
[doc] PR for Add YBA tab and info under YugabyteDB > Explorer > Continuous Availability

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-6986

All the structure and text is done.

To Do: Run the YB Simulator Workload app (see https://docs.yugabyte.com/preview/explore/#set-up-yugabytedb-universe for setup instructions) on a universe in YBA, take a few screenshots from the YBA UI (optionally) and a few screenshots from the YB Simulator app UI. Then add the screenshots to replace the ones commented out in macos-yba.md

Note: I have commented out two images which I will uncomment after the Horizontal scalability [PR](https://github.com/yugabyte/yugabyte-db/pull/15817) is merged.

### CODE REVIEW

File: docs/content/preview/explore/fault-tolerance/macos-yba.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  </li>
+  <li>
+    <a href="../macos-yba/" class="nav-link active">
+      <img src="/icons/server.svg" alt="Server Icon">
+      YugabyteDB Anywhere
+    </a>
+  </li>
+</ul>
+
+Follow the [setup instructions](../../#set-up-yugabytedb-universe) to start a single region three-node universe in YugabyteDB Anywhere, connect the [YB Workload Simulator](../../#set-up-yb-workload-similator) application, and run a read-write workload. To verify that the application is running correctly, navigate to the application UI at <http://localhost:8080/> to view the universe network diagram, as well as Latency and Throughput charts for the running workload.
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos-yba.md:61]: typo in Workload app link

File: docs/content/preview/explore/fault-tolerance/macos-yba.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+1. Navigate to **Universes** and select your universe.
+
+1. Select **Nodes** to view the total read and write IOPS per node and other statistics, as shown in the following illustration:
+
+  <!-- ![Read and write IOPS with 3 nodes](/images/ce/transactions_anywhere_observe1.png) -->
+
+   Notice that both the reads and the writes are approximately the same across all nodes, indicating uniform load.
+
+1. Select **Metrics** to view charts such as YSQL operations per second and latency, as shown in the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos-yba.md:75]: numbering restarts - fix indenting

File: docs/content/preview/explore/fault-tolerance/macos-yba.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+   <!-- ![Performance charts for 3 nodes](/images/ce/transactions_anywhere_chart.png) -->
+
+1. Navigate to the [YB Workload Simulator application UI](http://127.0.0.1:8000/) to view the latency and throughput on the universe while the workload is running, as per the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos-yba.md:79]: 8080

File: docs/content/preview/explore/fault-tolerance/macos-yba.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+1. Verify that the details by selecting **Nodes**. Expect to see that the load has been moved off the stopped node and redistributed to the remaining nodes, as shown in the following illustration:
+
+   ![Read and write IOPS with one node stopped](/images/ce/stop-node-yba.png)
+
+1. Navigate to **Metrics** to observe a slight spike and drop in the latency and YSQL Ops / Sec charts when the node is stopped, as shown in the following illustration:
+
+   ![Performance metrics with a node dead](/images/ce/stop-node-chart-yba.png)
+
+Alternatively, you can navigate to the [YB Workload Simulator application UI](http://127.0.0.1:8000/) to see the node being removed from the network diagram when it is stopped (it may take a few minutes to display the updated network diagram). Also notice a slight spike and drop in the latency and throughput, both of which resume immediately, as shown in the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos-yba.md:99]: 8080 port

File: docs/content/preview/explore/fault-tolerance/macos.md [hunk 5]
```diff
@@ -18,64 +18,88 @@ menu:
...
 
-To view a table of per-node statistics for the cluster, navigate to the [tablet-servers](http://127.0.0.1:7000/tablet-servers) page. The following illustration shows the total read and write IOPS per node. Note that both the reads and the writes are roughly the same across all the nodes, indicating uniform load across the nodes.
+To view a table of per-node statistics for the universe, navigate to the [tablet-servers](http://127.0.0.1:7000/tablet-servers) page. The following illustration shows the total read and write IOPS per node:
 
 ![Read and write IOPS with 3 nodes](/images/ce/fault-tolerance-evenly-distributed.png)
 
-To view the latency and throughput on the cluster while the workload is running, navigate to the [simulation application UI](http://127.0.0.1:8000/).
+Notice that both the reads and the writes are approximately the same across all nodes, indicating uniform load.
+
+To view the latency and throughput on the universe while the workload is running, navigate to the [simulation application UI](http://127.0.0.1:8000/), as per the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos.md:72]: port 8080

File: docs/content/preview/explore/fault-tolerance/macos.md [hunk 6]
```diff
@@ -18,64 +18,88 @@ menu:
...
+The `Time since heartbeat` value for that node starts to increase. When that number reaches 60s (1 minute), YugabyteDB changes the status of that node from ALIVE to DEAD. Observe the load (tablets) and IOPS getting moved off the removed node and redistributed to the other nodes, as per the following illustration:
 
 ![Read and write IOPS with one node stopped](/images/ce/fault-tolerance-dead-node.png)
 
 With the loss of the node, which also represents the loss of an entire fault domain, the cluster is now in an under-replicated state.
 
-Navigate to the [simulation application UI](http://127.0.0.1:8000/) to see the node removed from the network diagram when it is stopped. Note that it may take about 60s (1 minute) to display the updated network diagram. You can also notice a spike and drop in the latency and throughput, both of which resume immediately.
+Navigate to the [simulation application UI](http://127.0.0.1:8000/) to see the node removed from the network diagram when it is stopped, as per the following illustration:
```
Comment (ddhodge) [inline docs/content/preview/explore/fault-tolerance/macos.md:92]: port 8080

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/explore/fault-tolerance/macos-yba.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos-yba.md:61]: fixed

File: docs/content/preview/explore/fault-tolerance/macos-yba.md (hunk 2 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos-yba.md:75]: fixed

File: docs/content/preview/explore/fault-tolerance/macos-yba.md (hunk 3 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos-yba.md:79]: fixed

File: docs/content/preview/explore/fault-tolerance/macos-yba.md (hunk 4 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos-yba.md:99]: fixed

File: docs/content/preview/explore/fault-tolerance/macos.md (hunk 5 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos.md:72]: fixed

File: docs/content/preview/explore/fault-tolerance/macos.md (hunk 6 above)
Comment (lizayugabyte) [reply docs/content/preview/explore/fault-tolerance/macos.md:92]: fixed

**Pull request merged**